test/flattenDepth.js

```diff
@@ -0,0 +1,23 @@
+import * as RA from '../src';
+import eq from './shared/eq';
+
```

Comment:
I'll get to tests Tomorrow. They assertions are fine we just need to fine tune it a little.

Is iterable is a good example how we want to make test recently here: https://github.com/char0n/ramda-adjunct/blob/master/test/isIterable.js